webservices_dispatch_action/env_management.py

```diff
@@ -13,21 +13,21 @@ class SensitiveEnv:
     ]
 
     def __init__(self):
-        self.clasified_info = {}
+        self.classified_info = {}
 
     def hide_env_vars(self):
         """Remove sensitive env vars"""
-        self.clasified_info.update(
+        self.classified_info.update(
             {
-                k: os.environ.pop(k, self.clasified_info.get(k, None))
```

Comment:
One of the automated tools changed this typo - but not sure if that is potentially used externally anywhere.